_episodes/14-modules.md

```diff
@@ -247,6 +247,7 @@ Command: seq       common transformation of FASTA/Q
          randbase  choose a random base from hets
          cutN      cut sequence at long N
          listhet   extract the position of each het
+         hpc       homopolyer-compressed sequence
```

Comment:
polymer

Reply:
That is probably correct, but I have pasted it in exactly as it appeared on the screen.